PWGJE/Core/JetTaggingUtilities.h

```diff
@@ -284,7 +333,7 @@ int jetOrigin(T const& jet, U const& particles, float dRMax = 0.25)
   bool firstPartonFound = false;
   typename U::iterator parton1;
```

Comment:
in line 304 check if the iterator on particles changes if particles are sliced by collision or not

Reply:
if I used particles per collision like `auto const particlesPerColl = particles.sliceBy(particlesPerCollision, collision.mcCollisionId());`
it doesn't work to use iterator.
```
error: no type named 'iterator' in 'o2::framework::PresliceBase<o2::soa::             Table<o2::aod::Hash<"JMcParticles"_h>, o2::aod::Hash<"JMCPARTICLE/0"_h>, o2::aod::Hash<"AOD"_h>>>'
   8   typename U::iterator hfparticle;
```